### DISCUSSION TITLE
HDHomeRun, Windows setup configuration with raw/direct stream and reduced quality stream?

### DISCUSSION THREAD

**Discussion by wazerstar** [bluenviron/mediamtx#899] [Questions] [answered]
Hi I recently found this git and it seems very interesting, however I cannot understand how a setup would be and how it would look for configuration on a windows machine with 3-4 hdhomeruns being ran on same netwot and pinpointing to all channels found via hdhomerun unit to this software.

hdhomerun runs on IP:5004 -

An internal example for a channel would be http://192.168.1.5:5004/a/1 which would take the number 1 as channel number, the streams is direct mpeg-ts streams

Could I get some guidelines and examples of configuration for unmodded stream and reduced stream quality in order to save bandwidth via mobile etc.

Thanks.

**Comment by aler9** [comment, marked as answer]
Hello,

> the streams is direct mpeg-ts streams

At the moment we don't support proxing MPEG-TS streams directly through the server, but you can use ffmpeg to publish the stream from hdhomerun to the server:

```
ffmpeg -i http://192.168.1.5:5004/a/1 -c copy -f rtsp rtsp://localhost:8554/a1
```

> Could I get some guidelines and examples of configuration for unmodded stream and reduced stream quality in order to save bandwidth via mobile

After you're able to publish the streams to the server, you can create some secondary streams with reduced quality, by following instructions here:

https://github.com/aler9/rtsp-simple-server#remuxing-re-encoding-compression

Please keep in mind that each re-encoding operation consumes a lot of CPU. When i have bandwidth problems, i usually tune the original stream instead of re-encoding.

[a comment by an automated account is not shown]
